.github/workflows/pr-env-destroy.yml

```diff
@@ -84,6 +84,7 @@ jobs:
           terraform -chdir=terraform/infrastructure destroy \
             --var-file=etc/dev.tfvars \
             --var assume_account=${{ steps.get_account_id.outputs.aws_account_id }} \
```

Comment:
I think you only need the `role_arn` var now so `-var assume_account=` and `-var assume_role=` can be removed also. However, not needed for the fix the to work, it's just tidy up.

Reply:
removed